### PR TITLE
Turkish Translation Added

### PR DESCRIPTION
**Pull request by muklavon**
Okay, new pull request because I ruined the previous one somehow.

I translated Haveno completely into Turkish, did a test myself and almost everything works as it should. There are just some places where the sentences don't quite fit on the screen. I will solve this.

The problem is that when I put my Turkish translation in "core/src/main/resources/i18n" and install it, Haveno doesn't show a new language option called Turkish in the language settings menu.

When I did my own test, I deleted the entire contents of the German language option and replaced it with my Turkish translation using the same filename. When I selected it in the language menu, my translation worked flawlessly.

If anyone can do it, I would like to ask you to solve this problem with the language menu because the translation is completely finished and working. You just need to adjust the language menu so it can show Turkish language as a option.

I will upload photos to show how the UI looks like

VirtualBox_Windows_20_06_2024_15_07_26
VirtualBox_Windows_20_06_2024_15_07_47
VirtualBox_Windows_20_06_2024_15_08_08
VirtualBox_Windows_20_06_2024_15_08_39

### CODE REVIEW

**Comment by woodser** [comment]
Please enable the Turkish language by moving it from the commented section to the uncommented section here: https://github.com/haveno-dex/haveno/blob/1f0e326a16827f849f42408b847ae27edd758494/core/src/main/java/haveno/core/locale/LanguageUtil.java#L47

Also ensure the translations are not too long to appear in the UI. Better to reword things to be shorter in that case.

**Comment by woodser** [comment]
Also here's a good place to chat back and forth since DM isn't working.

**Comment by muklavon** [comment]
> Please enable the Turkish language by moving it from the commented section to the uncommented section here:
 https://github.com/haveno-dex/haveno/blob/1f0e326a16827f849f42408b847ae27edd758494/core/src/main/java/haveno/core/locale/LanguageUtil.java#L47

+Done, I moved it to uncommented section. I will update my Haveno and see if it works after making some alignments about long sentences not fitting the screen

> Also ensure the translations are not too long to appear in the UI. Better to reword things to be shorter in that case.

+Sure, I will take a look at it.

**Comment by muklavon** [comment]
Okay. Now Turkish language option is available on the menu and all sentences fit on screen.

It is ready to merge :)

**Comment by woodser** [comment]
Have you also tested the flow to complete a trade?

**Comment by muklavon** [comment]
> Have you also tested the flow to complete a trade?

Yes, I did now. In this test, I used a virtual windows machine as stagenet user 1 and 2. I created a face to face Monero to Turkish Lira exchange and screenshot all steps. First I will provide point of view for user 1 and then user 2.

Stagenet User 1:



eno-dex/haveno/assets/173

062365/3076f519-6bd7-494a-93f4-8e6b03a8cf4a)

Stagenet User 2:




Trade was perfectly successful. I tested everything and made some minor changes about translation. I think it is ready to merge, You can try it yourself too

**Review by woodser — APPROVED** [review]

File: core/src/main/resources/i18n/displayStrings_tr.properties [hunk 1]
```diff
@@ -0,0 +1,3345 @@
...
+mainView.footer.usingTor=(Tor üzerinden)
+mainView.footer.localhostMoneroNode=(localhost)
+mainView.footer.clearnet=(clearnet üzerinden)
+mainView.footer.xmrInfo={0} {1}
+mainView.footer.xmrFeeRate=/ Ücret oranı: {0} sat/vB
+mainView.footer.xmrInfo.initializing=Haveno ağına bağlanıyor
+mainView.footer.xmrInfo.synchronizingWith={0} ile senkronize ediliyor blok: {1} / {2}
+mainView.footer.xmrInfo.connectedTo={0}'a bağlandı blok {1}
+mainView.footer.xmrInfo.synchronizingWalletWith={0} ile cüzdan senkronize ediliyor blok: {1} / {2}
+mainView.footer.xmrInfo.syncedWith={0} ile senkronize edildi blok {1}
```
Comment (HardenedSteel) [inline core/src/main/resources/i18n/displayStrings_tr.properties:276]: In your screenshot it says: ``Monero Stagenet (Tor üzerinden) ile senkonize edildi`` but instead it should be ``Monero Stagenet ile (Tor üzerinden) senkonize edildi``

File: core/src/main/resources/i18n/displayStrings_tr.properties [hunk 2]
```diff
@@ -0,0 +1,3345 @@
...
+
+tradeFeedbackWindow.title=Ticaretiniz başarıyla tamamlandı!
+tradeFeedbackWindow.msg.part1=Deneyiminiz hakkında geri bildirim almak isteriz. Yazılımı geliştirmemize ve pürüzleri gidermemize yardımcı olacaktır.
+tradeFeedbackWindow.msg.part2=Herhangi bir sorunuz varsa veya herhangi bir sorun yaşadıysanız, diğer kullanıcılar ve katkıda bulunanlarla Matrix sohbet odamızda iletişime geçin:
+tradeFeedbackWindow.msg.part3=Haveno'yu kullandığınız için teşekkürler!
+
+portfolio.pending.role=Rolüm
+portfolio.pending.tradeInformation=Ticaret bilgileri
+portfolio.pending.remainingTime=Kalan süre
+portfolio.pending.remainingTimeDetail={0} (kadar {1})
```
Comment (HardenedSteel) [inline core/src/main/resources/i18n/displayStrings_tr.properties:867]: it should be ``{0} ({1} tarihine kadar)``

**Review by HardenedSteel — COMMENTED** [review]
Just checked your screenshots, didn't check the all strings. There's incorrect translations.

File: core/src/main/resources/i18n/displayStrings_tr.properties (hunk 2 above)
Comment (muklavon) [reply core/src/main/resources/i18n/displayStrings_tr.properties:867]: thanks, I made mistakes in some places, I would appreciate if you share more translation mistakes below this post. I will collect them all over time and open a new pull request

**Pull request merged**